rpc/rpc_client.go

```diff
@@ -639,6 +639,22 @@ func (c *client) PutDeploy(ctx context.Context, deploy types.Deploy) (PutDeployR
 	return result, nil
 }
 
+func (c *client) PutTransaction(ctx context.Context, transaction types.TransactionV1) (PutTransactionResult, error) {
```

Comment:
if in a future version we will allow PutTransaction with a Deploy object, I would consider a different name for PutTransaction now. like PutTransactionV1.